### ISSUE TITLE
Installation behaviour for updated versions

### ISSUE THREAD

**Issue by bernt-matthias** [galaxyproject/ansible-galaxy-tools#46]
The problem appeared for hisat version 16 which was an update of version 15 (https://github.com/galaxyproject/tools-iuc/issues/1514). I had 15 installed and tried to install also 16. The log output was: 

```
Skipping installation of revision 21e8ea8e1adb of repository 'hisat2' because it was installed with th
e (possibly updated) revision d5fe9aead222 and its current installation status is 'Installed'.
```

The ansible script returned no error. I would expect some error or that the tool gets installed..?

**Comment by bernt-matthias** [comment]
Adding to this I would like to ask if it is possible to specify the versions instead of the revisions that should be installed?

**Comment by mvdbeek** [comment]
Yeah, so for the moment you need to specify the revision you would like to install if you want to update. Specifying revision numbers would be great, but needs to be implemented in ephemeris, as well as installing the latest revision.
